### DISCUSSION TITLE
Previous/next links "in this file" go back to code search home

### DISCUSSION THREAD

**Discussion by instanceofme** [community/community#9813] [Other Feature Feedback, Questions, & Ideas] [answered]
Example: searching `SparkSession` and going to [this result page](https://cs.github.com/apache/spark/blob/aaa349de3ab4d03eebc692c9c577157e996c8f90/python/pyspark/sql/session.py?q=SparkSession#L107), then clicking in the right-hand popup the `previous` or `next` link to see the next occurrence in the file takes me back instead to CS home.

![Screen Shot 2022-01-07 at 17 40 01](https://user-images.githubusercontent.com/402483/148578685-707fa64c-2765-4b57-8ce6-aa1a7259c9d2.png)

**Comment by colinwm** [comment, marked as answer]
Sorry about that! This is the same as https://github.com/github/feedback/discussions/9743, which should be fixed now.
